charts/rmd/templates/deployment-rmd.yaml

```
@@ -23,7 +23,11 @@ spec:
             {{- include "rmd.djangoDatabasePassword" . | nindent 12 }}
           envFrom:
             - secretRef:
+                {{- if .Values.rmd.settings.emailPassword }}
+                name: {{ .Values.rmd.settings.emailPassword }}
+                {{- else }}
```

it should probably look something like this:
```suggestion
                {{- if .Values.rmd.existingSecret }}
                name: {{ .Values.rmd.existingSecret }}
                {{- else }}
```
There is a bunch of other `secretRef` things that probably also need changing (in the dbmigrate and cronjob pods)